docker-compose.yml.development

```diff
@@ -0,0 +1,51 @@
+# This file is configured for deployment of the CantusDB project in local development.
+# When building the project locally, replace the contents of docker-compose.yml with the
+# contents of this file.
+
+version: '3'
+
+services:
+  django:
+    build:
+      context: ./django
+    volumes:
+      - ./:/code/
+      - static_volume:/resources/static
+      - media_volume:/resources/media
+      - api_cache_volume:/resources/api_cache
+    env_file: ./config/envs/dev_env
+    ports:
+      - 3000:3000
```

Comment:
I was curious too why this port mapping was here and if my code working was successful, it's related to VSCode....in which case it should only be for development, I think

Reply:
good catch! I'll make this change.